src/cmd-buildextend-metal

```diff
@@ -274,10 +274,16 @@ platforms_json="${tmp_builddir}/platforms.json"
 yaml2json "${configdir}/platforms.yaml" "${platforms_json}"
 
 # Currently we only support OSBuild for qemu and metal disk images
+osbuild_extra_args=()
+osbuild_mpp_config="coreos.osbuild.${basearch}.mpp.yaml"
 if [ "${image_type}" == "qemu" ] || [ "${image_type}" == "metal" ] || [ "${image_type}" == "metal4k" ]; then
     # Right now we don't have support fully fleshed out for secex on s390x
     if [[ $secure_execution -ne "1" ]]; then
         OSBUILD_SUPPORTED=1
+    elif [[ -f "${hostkey}" ]]; then
+        OSBUILD_SUPPORTED=1
+        osbuild_mpp_config="coreos.osbuild.s390x.secex.mpp.yaml"
+        osbuild_extra_args+=("--pubkey" "${ignition_pubkey}")
```

Comment:
I'm not that knowledgeable about how osbuild works and is implemented, but shouldn't the hostkey also be added here somewhere? Otherwise how does it know where the hostkey is for signing later?

Reply:
I think you're right. It looks like the template is assuming it's at `/srv/secex-hostkey`?